libs/parallelism/algorithms/tests/regressions/reduce_3641.cpp

```diff
@@ -13,18 +13,16 @@
 #include <hpx/modules/testing.hpp>
 #include "iter_sent.hpp"
 
-#include <cstddef>
 #include <cstdint>
-#include <iterator>
 
 int main()
 {
     std::int64_t result = hpx::ranges::reduce(hpx::execution::seq,
-        Iterator<std::int64_t>{0}, Sentinel<int64_t>{100}, std::int64_t(0));
+        iterator<std::int64_t>{0}, Sentinel<int64_t>{100}, std::int64_t(0));
```

Comment:
Please check that the files you've changed build for you:
```suggestion
        iterator<std::int64_t>{0}, sentinel<int64_t>{100}, std::int64_t(0));
```

Reply:
Sorry about that.